test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -21,14 +19,17 @@ public class DatabaseLoaderTests : BaseTestClass
         public DatabaseLoaderTests(ITestOutputHelper output)
             : base(output)
         {
+            DbProviderFactories.RegisterFactory("System.Data.SqlClient", typeof(System.Data.SqlClient.SqlClientFactory));
         }
 
         [LightGBMFact]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Can we open an issue to track enabling these tests on unix?

Reply:
Logged https://github.com/dotnet/machinelearning/issues/4156